bot-github-core/src/main/java/org/commonhaus/automation/github/context/BotComment.java

```diff
@@ -0,0 +1,78 @@
+package org.commonhaus.automation.github.context;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class BotComment {
+    final Pattern botCommentPattern;
+    final private String itemId;
+    private final String id;
+    private final int databaseId;
+    private final String url;
+    private String bodyString;
+
+    BotComment(Pattern botCommentPattern, String itemId, DataCommonComment comment) {
```

Comment:
Package private visibility in fields and constructor in public class with public methods. Yuck 😛

Reply:
I love package private fields. They are my favorite. ;)

Some things are best known w/in a package (not w/in an inheritance tree)